zbus/src/message/fields.rs

```diff
@@ -14,7 +14,7 @@ use crate::message::{FieldCode, Header, Message};
 /// [`Field`]: enum.Field.html
 #[derive(Debug, Default, Clone, Type)]
 #[zvariant(signature = "a(yv)")]
-pub(crate) struct Fields<'f> {
+pub struct Fields<'f> {
```

Comment:
This used to be public but since it's very low-level API, I was glad when I was able to hide it. As I wrote on #1317, I'll need a compelling use case for this.